src/main/java/com/ibm/disni/verbs/SVCPostSend.java

```diff
@@ -57,7 +57,13 @@
 		 * @param send_flags new flags
 		 */		
 		void setSend_flags(int send_flags);
-		
+
+		/**
+		 * Modify the Opcode
+		 *
+		 * @param opcode the new opcode
+		 */
+		void setOpcode(int opcode);
```

Comment:
setting the opcode is already offered through the SendWRMod interface.

Reply:
Original SendWRMOd did not has setOpcode(). Please check. And my commit does add setOpcode() to SendWRMod. Please note that the modification located in the definition of SendWRMod.